tests/trainer/test_trainer.py

```diff
@@ -2565,6 +2588,37 @@ def test_accelerator_config_from_dataclass(self):
             self.assertEqual(trainer.accelerator.even_batches, False)
             self.assertEqual(trainer.accelerator.use_seedable_sampler, False)
 
+    @unittest.skipUnless(
+        FLAG_ACCELERATOR_SUPPORT_GRAD_ACCUM_KWARGS, "setting of gradient_accumulation_kwargs not supported"
+    )
```

Comment:
We have a `requires` instead. 

Let's change `src/transformers/testing_utils/require_accelerate` to accept a `min_version`, which defaults to `ACCELERATE_MIN_VERSION`

Reply:
sure. I followed the [`require_fsdp` function](https://github.com/huggingface/transformers/blob/d522afea1324b8156c929f3896df14762c9ea716/src/transformers/testing_utils.py#L364), and the style to [call a `partial` to bind the `min_version`](https://github.com/huggingface/transformers/blob/d522afea1324b8156c929f3896df14762c9ea716/tests/fsdp/test_fsdp.py#L90) before decorating the test function